JsonParser.cpp

```diff
@@ -0,0 +1,73 @@
+#include "JsonParser.h"
+
+std::map<std::string, std::string> JsonParser::parseJSON(std::istream& data){
+  std::string toParse;
+  std::string line;
+  while(std::getline(data, line)){
+    toParse += line;
+  }
+
+  return JsonParser::parseJSON(toParse);
+}
+
+std::map<std::string, std::string> JsonParser::parseJSON(const std::string& data){
+  try{
+    std::map<std::string, std::string> toReturn;
+    std::ifstream test(data);
+    std::string toParse = "";
+
+    if(test){
+      std::string line;
+      while(std::getline(test, line)){
+        toParse += line + "\n";
+      }
+    }else{
+      toParse = data;
+    }
+    toParse.erase(remove_if(toParse.begin(), toParse.end(), isspace), toParse.end());
```

Comment:
Akkor a karakter nevéből is törölni fogjuk a szóközöket?

Reply:
Megoldva, az új verzióban már a karakter nevében lehet whitespace.